src/items.cpp

```diff
@@ -233,23 +231,22 @@ Item* newItem(ItemType type, Status status, Sint16 beatitude, Sint16 count, Uint
 		{
 			if ( inventory == &stats[clientnum]->inventory )
 			{
-				int c;
```

Comment:
Not a fan of refactoring hotbar[] to auto - we will be revisiting the hotbar code for splitscreen so I'd rather leave it all in the same intact way. I have manually changed it back after the charclass PR

Reply:
Fixed in f76c8571